setup.py

```diff
@@ -1,10 +1,38 @@
+import pathlib
+import re
+
 from setuptools import setup
 
-VERSION = "0.0.1"
+VERSIONFILE = "carla/_version.py"
+verstrline = open(VERSIONFILE, "rt").read()
+VSRE = r"^__version__ = ['\"]([^'\"]*)['\"]"
+mo = re.search(VSRE, verstrline, re.M)
+if mo:
+    VERSION = mo.group(1)
+else:
+    raise RuntimeError("Unable to find version string in %s." % (VERSIONFILE,))
+
+# The directory containing this file
+HERE = pathlib.Path(__file__).parent
+
+# The text of the README file
+README = (HERE / "README.md").read_text()
 
 setup(
-    name="carla",
+    name="carla-recourse",
     version=VERSION,
+    description="A library for counterfactual recourse",
+    long_description=README,
+    long_description_content_type="text/markdown",
+    url="https://github.com/indyfree/CARLA",
+    author="Martin Pawelczyk, Sascha Bielawski, Joannes van den Heuvel, Tobias Richter and Gjergji Kasneci",
```

Comment:
Not sure if we need to put all the paper authors here, Gjergji didn't really contribute to the package